tools/govulncheck-parser/main.go

```
@@ -0,0 +1,104 @@
+		}
+	}
+
+	if len(osvIDsForCVE) > 0 {
+		return ResultNotReachable, nil
+	}
+
+	return ResultNotFound, nil
+}
+
+func main() {
+	if len(os.Args) != 2 {
```

golangci-lint (2.12.2)

[error] Magic number: 2, in <condition> detected

(mnd)